app/controllers/my_modules_controller.rb

```diff
@@ -341,6 +341,7 @@ def load_vars
     if @my_module
       @experiment = @my_module.experiment
       @project = @my_module.experiment.project if @experiment
+      @protocol = @my_module.protocol
```

Comment:
I suggest you remove this line, and handle it differently in the protocol widget (see comment there).